priv/repo/migrations/20160913080731_update_team_table.exs

```diff
@@ -0,0 +1,12 @@
+defmodule EspiDni.Repo.Migrations.UpdateTeamTable do
+  use Ecto.Migration
+
+  def change do
+    rename table(:teams), :token, to: :slack_token
+
+    alter table(:teams) do
+      add :google_token, :string
+      add :google_refresh_token, :string
+    end
+  end
```

Comment:
I have the feeling that this should belong into 2 separate migrations. Because in case you have to rollback the rename action, you will also lose all the google tokens, right?